apps/dashboard/src/components/billing/plan.tsx

```diff
@@ -218,6 +218,12 @@ const buildFeatureArray: (columns: ApiServiceLevelEnum[]) => Feature[] = (column
         isBoolean: true,
       }),
     },
+    {
+      label: 'Notification snooze',
```

Comment:
Not sure about the copy here - whats more recognizable "Schedule reminders" maybe?

Reply:
`Remind me later` functionality